docs/docs/dev_docs/cli/updating.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: Updating
+title: Staying Updated
 ---
 
 There are 4 components whose versions need to be kept compatible:
+
```

Comment:
Can we create a short list of things to do at the top of the doc, in 4 bullet points to address [this](https://github.com/AztecProtocol/aztec-packages/issues/2984) issue?

Something like:
1. Update the sandbox with 
```
/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
```
2. Update the CLI with
```
npm i -g @aztec/cli
```
3. Update Noir compiler to:
<InstallNargoInstructions />  <---(this will always be the lastest version)
4. Update aztec-nr packages

```toml
#nargo.toml
[dependencies]
aztec = { git="https://github.com/AztecProtocol/aztec-packages", tag="#include_aztec_version", directory="yarn-project/aztec-nr/aztec" }
value_note = { git="https://github.com/AztecProtocol/aztec-packages", tag="#include_aztec_version", directory="yarn-project/aztec-nr/value-note" }
```
^ the macro will always make it the correct version

Reply:
The rest of the doc can include the extra detail, but this keeps the key info at the top of the page